client/views/admin/apps/AppDetailsHeader.tsx

```diff
@@ -0,0 +1,93 @@
+import { AnimatedVisibility, Box, Icon, PositionAnimated, Tooltip } from '@rocket.chat/fuselage';
+import { formatDistanceStrict } from 'date-fns';
+import React, { FC, RefObject, useRef, useState } from 'react';
+
+import AppAvatar from '../../../components/avatar/AppAvatar';
+import { useTranslation } from '../../../contexts/TranslationContext';
+import AppMenu from './AppMenu';
+import AppStatus from './AppStatus';
+import { App } from './types';
+
+type AppDetailsPageHeaderProps = {
+	app: App;
+};
+
+const AppDetailsHeader: FC<AppDetailsPageHeaderProps> = ({ app }) => {
+	const t = useTranslation();
+	const bundleRef = useRef<Element>();
+	const [isHovered, setIsHovered] = useState(false);
+
+	const { iconFileData = '', name, author, version, iconFileContent, installed, modifiedAt, bundledIn, description } = app;
+
+	const lastUpdated = formatDistanceStrict(new Date(modifiedAt), new Date(), { addSuffix: false });
+
+	return (
+		<Box display='flex' flexDirection='row' mbe='x20' w='full'>
+			<AppAvatar size='x124' mie='x20' iconFileContent={iconFileContent} iconFileData={iconFileData} />
+			<Box display='flex' flexDirection='column'>
+				<Box display='flex' flexDirection='row' alignItems='center' mbe='x8'>
+					<Box fontScale='h1' mie='x8'>
+						{name}
+					</Box>
+					{Boolean(bundledIn.length) &&
+						bundledIn.map((bundle) => (
```

Comment:
I suggest breaking the content of this map into a new component to make easy the manutenabillity.

Reply:
Solved